docs/output.md

```diff
@@ -306,6 +306,17 @@ print(repr(result.output))
 
 _(This example is complete, it can be run "as is")_
 
+#### Handling Multiple Tool Calls
+
+When the model calls multiple tools in a final response, you can control how tool calls are executed with the `end_strategy`:
+
+- `'early'` (default): Tools are executed in order until a valid final result is found, then remaining tool calls are skipped
```

Comment:
This makes it seem like `random_tool, final_result, random_tool` will have `random_tool` executed first, but we do in fact run output tools first